server/planning/planning.py

```diff
@@ -252,6 +252,6 @@ class PlanningResource(superdesk.Resource):
     resource_methods = ['GET', 'POST']
     item_methods = ['GET', 'PATCH', 'PUT', 'DELETE']
     public_methods = ['GET']
-    privileges = {'POST': 'planning',
-                  'PATCH': 'planning',
+    privileges = {'POST': 'planning_planning_management',
+                  'PATCH': 'planning_planning_management',
                   'DELETE': 'planning'}
```

Comment:
Because the delete will be changed to a spike functionality in a coming pull request for event, planning and agenda items.